components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -76,40 +76,46 @@
               *ngIf="(updateServerForm.get('ip_address').hasError('required') || updateServerForm.get('ip_address').hasError('pattern')) && updateServerForm.get('ip_address').dirty">
               IP Address is required.
             </chef-error>
-          </chef-form-field>  
+          </chef-form-field>
           <div id="button-bar">
             <chef-button [disabled]="isLoading || !updateServerForm.valid || !updateServerForm.dirty"
-              primary inline (click)="saveServer()">                
-            <chef-loading-spinner *ngIf="saveInProgress"></chef-loading-spinner>
-            <span *ngIf="saveInProgress">Saving...</span>
-            <span *ngIf="!saveInProgress">Save</span></chef-button>
+            primary inline (click)="saveServer()">
+              <chef-loading-spinner *ngIf="saveInProgress"></chef-loading-spinner>
+              <span *ngIf="saveInProgress">Saving...</span>
+              <span *ngIf="!saveInProgress">Save</span></chef-button>
             <span id="saved-note" *ngIf="saveSuccessful && !updateServerForm.dirty">All changes saved.</span>
           </div>
         </form>
       </section>
       <section class="page-body" *ngIf="tabValue === 'orgs'">
-        <ng-container>
+        <chef-loading-spinner *ngIf="orgsListLoading" size="50"></chef-loading-spinner>
+        <ng-container *ngIf="!orgsListLoading">
           <chef-toolbar>
-            <chef-button id="create-button" primary (click)="openCreateModal('create')">Add Org</chef-button>
+            <app-authorized [allOf]="['/api/v0/infra/servers', 'post']">
+              <chef-button id="create-button" primary (click)="openCreateModal('create')">Add Chef Organization
```

Comment:
Not introduced by you I realize, but please delete this unused parameter here and in the definition.

Also delete `modalType` on chef-server-details.component.ts:L52

```suggestion
              <chef-button id="create-button" primary (click)="openCreateModal()">Add Chef Organization
```